app/app.go

```diff
@@ -162,6 +162,7 @@ func getGovProposalHandlers() []govclient.ProposalHandler {
 		upgradeclient.ProposalHandler,
 		upgradeclient.CancelProposalHandler,
 		// this line is used by starport scaffolding # stargate/app/govProposalHandler
+		oracleclient.ProposalHandler,
```

Comment:
If it is not registered here, it will not be registered in the command below.
`panacead tx gov submit-proposal ...`